src/app/prepare-round-test/prepare-round-test.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { FirebaseRound } from '../round';
+import { FirebaseService } from '../firebase.service';
+import { Observable } from 'rxjs';
+import { Session } from '../session';
+
+@Component({
+  selector: 'app-prepare-round-test',
+  templateUrl: './prepare-round-test.component.html',
+  styleUrls: ['./prepare-round-test.component.scss']
+})
+export class PrepareRoundTestComponent implements OnInit {
+
+  userID: string;
+  testSession: string;
+  testRoundData: FirebaseRound;
+
+  constructor(public firebaseService: FirebaseService) {
+    this.testSession = 'kugTpWqJyrXaJZ4ZB6zE';
```

Comment:
Would it be possible to create this session if it doesn't already exist? (At the moment, you need to create it manually with the Firebase console.)